jest.config.cjs

```diff
@@ -5,12 +5,13 @@ module.exports = {
   testEnvironment: 'jsdom',
   testRegex: 'src/.+\\.(test|spec)\\.[jt]sx?$',
   transform: {
+    '\\.js$': 'babel-jest',
     '^.+\\.(t|j)sx?$': '@swc/jest',
   },
   moduleNameMapper: {
     '@qlik-trial/sprout/icons/.*': '<rootDir>/test-utils/sprout-icons-mock.tsx',
   },
   coveragePathIgnorePatterns: ['dist', 'lib', 'build-utils', './src/constants'],
-  transformIgnorePatterns: ['<rootDir>/node_modules/(?!(cliui|wrap-ansi)/*)'],
+  transformIgnorePatterns: ['node_modules/.pnpm/(?!(cliui|wrap-ansi|d3-color)@)'],
```

Comment:
Followed the pattern provided here https://jestjs.io/docs/configuration#transformignorepatterns-arraystring for pnpm